src/Content/Project/ContainerContent.module.css

```diff
@@ -0,0 +1,16 @@
+@media (min-width: 1000px) {
+  .container_content {
```

Comment:
Css class names are not consistent

In some places it is camelCase. Here it is underscore case